tox.ini

```diff
@@ -3,16 +3,19 @@ recreate = true
 envlist = py-unittest
 indexserver =
     preinstall = https://pypi.python.org/simple
+requires=tox-conda
```

Comment:
Ok, looking at the log files, it looks like the right version of python is picked up, so I guess that is fine.

Reply:
So in the 3.8 build (ubuntu) I see in the logs:
```
.tox/py-unittest/lib/python3.8/site-packages/glue/markup.py:7
  /home/runner/work/pycbc/pycbc/.tox/py-unittest/lib/python3.8/site-packages/glue/markup.py:7: DeprecationWarning: glue.markup is deprecated and will be removed in lscsoft-glue 4.0.0, please update your workflow to use the MarkupPy package.
    warnings.warn(
```

On the 3.9 build I see:

```
.tox/py-unittest/lib/python3.9/site-packages/glue/markup.py:7
  /home/runner/work/pycbc/pycbc/.tox/py-unittest/lib/python3.9/site-packages/glue/markup.py:7: DeprecationWarning: glue.markup is deprecated and will be removed in lscsoft-glue 4.0.0, please update your workflow to use the MarkupPy package.
    warnings.warn(
```

Is that sufficient? As far as I can see tox-conda should otherwise reproduce conda itself.